### PR TITLE
Add references to boundr and other relevant packages

### LINKED ISSUE #85: Your ONS API function
Hi there,

I was pointed to your ONS API functions today, nice work.
I just wanted to let you know that I've had a package available for a few years now that does some very similar things. In case it's useful to you or provides any functionality you don't yet have.

[boundr](https://github.com/francisbarton/boundr)
[pkgdown site](https://francisbarton.github.io/boundr/)

I'd be interested in any feedback you have for me on the UI and making the functions user-friendly. Your shorthand functions are a neat idea.
My main aim was to avoid the user having to go to the OG site and look up the table name they needed.

Please see what you think, and maybe let me know any ideas! Thanks.

### PR DESCRIPTION
**Pull request by cjrace**
# Brief overview of changes

Added references to the boundr package and a couple of other relevant ones on our README.

## Why are these changes being made?

1. We have a slight overlap with boundr, and should be encouraging use of that where we can to avoid duplication
2. dfeR is like our 'home' package in DfE that most analysts will use, so it's a handy place to highlight other relevant packages (this is also helpful for defining the scope of what dfeR will and won't do so it's not just open ended.

## Detailed description of changes

N / A

## Additional information for reviewers

More context can be found in the conversation on #85 

## Issue ticket number/s and link

Resolves #85

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ Where we can, we use their API to get the data, so that we have completely repro
...
 
-We have a `get_ons_api_data()` function that acts as a wrapper to the ONS API, it does things like converting readable parameters into a query string and also handles batching and multiple requests if needed, so you get all of the data in one nice neat data frame (there's a limit on the rows per single query for the API).
+We have a `get_ons_api_data()` function that acts as a wrapper to the ONS API, it does things like converting readable parameters into a query string and also handles batching and multiple requests if needed, so you get all of the data in one nice neat data frame (there's a limit on the rows per single query for the API). If you're looking to expand on this function at all, you should first check if the [boundr package](https://github.com/francisbarton/boundr) does what you need, as that gives a number of methods for extracting data from the portal as well.
```
Comment (jen-machin) [inline .github/CONTRIBUTING.md:278]: When would you suggest using our function vs using boundr?

**Review by jen-machin — COMMENTED** [review]
One minor suggestion!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (cjrace) [reply .github/CONTRIBUTING.md]: Generally use ours first as it's built for our needs in the package. Then check boundr if ours doesn't do what you need, and then if neither do make a suggestion on here if it's DfE specific or on boundr if more general / others might benefit too

Updated to put that into the text

**Comment by cjrace** [comment]
[dfeR: add references to boundr](https://trello.com/c/9MUu9V1M/1868-dfer-add-references-to-boundr)

**Review by jen-machin — APPROVED** [review]

**Pull request merged**